tests/unit/test_cors.py

```diff
@@ -2,22 +2,31 @@
 
 from localstack import config
 from localstack.aws.handlers import cors
+from localstack.config import HostAndPort
+
+# The default host depends on whether running in Docker (see config.py::default_ip) but that's good enough for testing:
+default_gateway_listen = [HostAndPort(host="0.0.0.0", port=4566)]
+default_gateway_listen_ext = [
+    HostAndPort(host="0.0.0.0", port=4566),
+    HostAndPort(host="0.0.0.0", port=443),
+]
```

Comment:
It feels a bit weird testing the -ext configuration here, but :shrug: it was here before... :joy:

Reply:
yeah, valid point. I would try to avoid localstack<=>localstack-ext refactorings within the scope of the release branch ...